demo_data/organizations.json

```diff
@@ -0,0 +1,48 @@
+{
+  "organizations": [
+    {
+      "users": [
+        {
+          "name": "fjelltopp_member",
+          "capacity": "member"
+        },
+        {
+          "name": "fjelltopp_editor",
+          "capacity": "editor"
+        },
+        {
+          "name": "fjelltopp_admin",
+          "capacity": "admin"
+        }
+      ],
+      "display_name": "Fjelltopp",
+      "description": "Fjelltopp demo organization",
+      "image_display_url": "",
```

Comment:
There's a fjelltopp logo already hosted at: http://ckan:5000/fjelltopp.png 

Maybe just add it here for visual guys like myself who will be bugged by the lack of image.